frontend/src/components/SignIn.js

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from "react";
+import { InputText } from "./InputText.js";
+
+export const SignIn = ({ setSignedIn }) => {
+  const [inputValue, setInputValue] = useState({
+    email: "",
+    password: "",
+  });
+  const [loginFailed, setLoginFailed] = useState(false);
+
+  const handleFormSubmit = (event) => {
+    event.preventDefault();
+    fetch(`https://project-auth-login.herokuapp.com/sessions`, {
+      method: "POST",
+      body: JSON.stringify({
+        email: inputValue.email,
+        password: inputValue.password,
+      }),
```

Comment:
Email and password seems to be all there is to inputValue so you could shorten it:
`body: JSON.stringify(inputValue)`